sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -1840,6 +1840,47 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(ArrayRepeat(Literal("hi"), Literal(null, IntegerType)), null)
   }
 
+  test("Array prepend") {
```

Comment:
```scala
- test("Array prepend") {
+ test("SPARK-41233: ArrayPrepend") {
```

Reply:
This is not a bug fix, also need to add jira number?